dict/portuguese.yml

```diff
@@ -2,290 +2,290 @@
+    'LV': [Letónia, Letônia, Letã*, Letões, Riga]
+    'NO': [Noruega, Norueguês*, Noruegues*, Oslo]
+    'SE': [Suécia, Suec*, Estocolmo]
     'SJ': [Svalbard e Jan Mayen, Norueguês*, Noruegues*]
 
   SOUTH:
-    'AD': [Andorra, Principado de Andorra, Principado dos Vales de Andorra, Andorran*, Andorrense*, Andorrian*, Andorrenh*]
-    'AL': [Albânia, República da Albânia, Albanês*, Tirana]
+    'AD': [Andorra, Andorran*, Andorrense*, Andorrian*, Andorrenh*]
+    'AL': [Albânia, Albanês*, Tirana]
     'BA': [Bósnia e Herzegovina, Bósnia, Bósni*, Sarajevo, Saraievo]
-    'ES': [Espanha, Reino de Espanha, Reino da Espanha, Espanhol*, Espanhóis, Madrid, Madri, Barcelona]
+    'ES': [Espanha, Espanhol*, Espanhóis, Madrid, Madri, Barcelona]
     'GI': [Gibraltar, Gibraltarin*, Gibraltin*, Llanit*, Llanis*]
     'GR': [Grécia, República Helênica, República Helénica, Grego*, Grega*, Atenas]
-    'HR': [Croácia, República da Croácia, Croata*, Zagreb, Zagrebe]
+    'HR': [Croácia, Croata*, Zagreb, Zagrebe]
     'IT': [Itália, República Italiana, Italian*, Roma]
     'KV': [Kosovo, Cossovo, Cosovo, Kossovo, Kosovar*, Cosovar*, Cossovar*, Pristina]
     'ME': [Montenegro, Montenegrin*, Podgoritza, Podgoriza, Podgorica]
-    'MK': [Macedônia do NORTH, Macedónia do NORTH, República da Macedônia do NORTH, República da Macedónia do NORTH, Macedóni*, Escópia, Skopje, Skopie]
-    'MT': [Malta, República de Malta, Maltês*, Valeta]
+    'MK': [Macedônia do Norte, Macedónia do Norte, Macedóni*, Escópia, Skopje, Skopie]
+    'MT': [Malta, Maltês*, Valeta]
     'PT': [Portugal, República Portuguesa, Português*, Portugues*, Lisboa]
-    'RS': [Sérvia, República da Sérvia*, Sérvi*, Belgrado]
-    'SI': [Eslovénia, Eslovênia, República da Eslovénia, Esloven*, Liubliana, Ljubljana]
-    'SM': [San Marino, Sereníssima República de San Marino, República de San Marino, São Marinho, São Marino, Samarinês*, São-Marinense*, São-Marinhense*, San-Marinense*]
-    'VA': [Vaticano, Cidade do Vaticano, Estado da Cidade do Vaticano, Vatican*]
+    'RS': [Sérvia, Sérvi*, Belgrado]
+    'SI': [Eslovénia, Eslovênia, Esloven*, Liubliana, Ljubljana]
+    'SM': [San Marino, São Marinho, São Marino, Samarinês*, São-Marinense*, São-Marinhense*, San-Marinense*]
+    'VA': [Vaticano, Vatican*]
 
   WEST:
-    'AT': [Áustria, República da Áustria, Austríac*, Viena]
-    'BE': [Bélgica, Reino da Bélgica, Belga*, Bélgic*, Bruxelas, Região da Capital Bruxelas]
-    'CH': [Suíça, Confederação Suíça, Suíç*, Zurique, Berna]
-    'DE': [Alemanha, República Federal da Alemanha, Alemã*, Berlim, Frankfurt am Main, Frankfurt sobre o Meno, Francoforte do Meno, Frankfurt, Francoforte]
+    'AT': [Áustria, Austríac*, Viena]
+    'BE': [Bélgica, Belga*, Bélgic*, Bruxelas, Região da Capital Bruxelas]
+    'CH': [Suíça, Suíç*, Zurique, Berna]
+    'DE': [Alemanha, Alemã*, Berlim, Frankfurt am Main, Frankfurt sobre o Meno, Francoforte do Meno, Frankfurt, Francoforte]
```

Comment:
I think it is enough to have only "Frankfurt".

Reply:
Ok. I'll adjust it.